recipe/meta.yaml

```diff
@@ -1,13 +1,15 @@
-{% set version = "0.1" %}
+{% set name = "osmnx" %}
+{% set version = "0.2.1" %}
+{% set md5 = "83614b39ca948bdbc1b4373b478d0372" %}
 
 package:
-  name: osmnx
+  name: {{ name }}
   version: {{ version }}
 
 source:
-  fn: osmnx-{{ version }}.tar.gz
-  url: https://github.com/gboeing/osmnx/archive/v{{ version }}.tar.gz
-  sha256: 9fda466d434022b55f41f0511c99cdf59bf5119630caa5263a315f2d3ab3524c
```

Comment:
> I switched to the PyPI tarball because the GitHub repo contains several examples and additional files that add up to 20+ MB. The PyPI tarball in contrast is only 34 KB. Perhaps I should be packaging or maintaining my repo in a different way, but it seemed inefficient to have users download 20+ MB to conda install the package when pip install only downloads 34 KB. Also, the PyPI tarball does contain the license file in this case. So, using the PyPI tarball seems to trade-off being "farther" from the source while being more lightweight. Is there a clear benefit one way or the other?

Nope. That is a good reason to use PyPI then :smile: 

> How much safer is sha256 and is it important to keep using it?

Probably not enough to be worth changing it again :wink: We can leave the `md5`

> Do you have any thoughts on their example and why they're doing it differently? Their way seems easier to maintain, but your way seems easier to grab the link to quickly check things.

Ironically I wrote most of that example :grimacing: Most people prefer all the values that needs updating to be at the top (like checksum and version). I prefer only the version.

Regarding the URL, the jinja vars with the name makes it easier to copy to other recipes, but I prefer to be able to copy-n-paste the URL as I do that more often then the former.

But that is really a personal choice, no big deal.

> Thanks for your comments. I'm looking forward to learning more about conda-forge as I figure out these processes!

This just came out today BTW: https://talkpython.fm/episodes/show/94/guarenteed-packages-via-conda-and-conda-forge